src/components/request/Request.tsx

```diff
@@ -48,33 +49,31 @@ const Request = () => {
     requestTxHash: requestTx,
   });
 
-  const isByTransactionParams = useIsByTransactionParams();
-  const { request, error } = useRequestParams(isByTransactionParams);
-
   useEffect(() => {
-    // TODO: would be nice to do something with the error here, like redirect to the homepage
-    if (!error && request) {
-      if (isByTransactionRequest(request)) {
-        setRequestId(
-          client.setActiveRequestByTransaction({
-            chainId: request.chainId,
-            transactionHash: request.transactionHash,
-            eventIndex: request.eventIndex ?? 0,
-          })
-        );
-      } else {
-        setRequestId(
-          client.setActiveRequest({
-            requester: request.requester.trim(),
-            identifier: request.identifier,
-            timestamp: request.timestamp,
-            ancillaryData: request.ancillaryData,
-            chainId: request.chainId,
-          })
-        );
-      }
+    const request: unknown = Object.fromEntries([...searchParams]);
```

Comment:
OOC, why does this need to be unknown?

Reply:
i guess it doesn't need to be, but when parsing the query params it expects to be passed in an unknown type. i mean technically it is unknown since anything can be set to the query params in the url